TI_Practice1/Validator.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using System.Globalization;
+using System.Text.RegularExpressions;
+
+namespace Practice1
+{
+    class Validator
+    {
+        private int _attempts;
+
+        public Validator(int attempts)
+        {
+            _attempts = attempts;
+        }
+
+        public double IsNumber()
+        {
+            int matchCount = 1;
+            double val = 0;
+            while (matchCount <= _attempts)
+            {
+                var input = Console.ReadLine();
+                input = input.Replace(',', '.');
+                //String pattern = @"^[1-9]\d*([\.|\,]\d+)?$$";
```

Comment:
We don't usually leave comments in code if they are not absolutely needed

Reply:
it is just another implementation because I do not understand the practice done criteria
So I decided to provide other option in the commented code